__tests__/app.tests.js

```diff
@@ -159,26 +159,91 @@ describe('GET /api/articles/:article_id/comments', () => {
     });
     test('200: Should return a message when there are no articles found for an existing article_id', () => {
         return request(app)
-        .get("/api/articles/7/comments")
+            .get("/api/articles/7/comments")
             .expect(200)
             .then((response) => {
                 expect(response.body.msg).toBe("No comments found for this article_id")
             });
     });
     test('404: Should return an error with a message when article_id does not exist.', () => {
         return request(app)
-        .get("/api/articles/9000/comments")
+            .get("/api/articles/9000/comments")
             .expect(404)
             .then((response) => {
                 expect(response.body.msg).toBe("Not Found: article_id does not exist.")
             });
     });
     test('400: Should return an error with a message when the article_id is invalid', () => {
         return request(app)
-        .get("/api/articles/bad_id/comments")
+            .get("/api/articles/bad_id/comments")
             .expect(400)
             .then((response) => {
-                expect(response.body.msg).toBe("Bad Request: Invalid article_id format")
+                expect(response.body.msg).toBe("Bad Request: Invalid article_id format.")
             });
     });
-});
\ No newline at end of file
+});
+
+describe('POST /api/articles/:article_id/comments', () => {
+    test('201: Should create and post a comment with 2 keys from the request body being username and body, including the other 3 keys', () => {
```

Comment:
could have another 201 that allows a comment with extra properties to go through (the extra properties would be ignored)